project/Dependencies.scala

```diff
@@ -46,17 +46,23 @@ object Dependencies {
   val sbtCompilerInterface = sbtOrg % "interface" % sbtMainVersion
   val sbtCompletion        = sbtOrg % "completion" % sbtMainVersion
 
+  val serializationVersion = "0.1.0-M1"
+  val serialization210 = "org.scala-sbt" % "serialization_2.10" % serializationVersion
+  val serialization211 = "org.scala-sbt" % "serialization_2.11" % serializationVersion
+  val serialization = "org.scala-sbt" %% "serialization" % serializationVersion
+
+  // These are needed for integration tests. Use the values used by sbt/serialization
   val picklingVersion = "0.10.0-M3"
   val pickling210 = "org.scala-lang.modules" % "scala-pickling_2.10" % picklingVersion
   val pickling211 = "org.scala-lang.modules" % "scala-pickling_2.11" % picklingVersion
   val pickling = "org.scala-lang.modules" %% "scala-pickling" % picklingVersion
 
+  // These are needed for integration tests. Use the values used by sbt/serialization
```

Comment:
kind of sucks that we have to keep these in sync with serialization :-/ no way to avoid? I guess ivy will pick the latest... ?